### PR TITLE
Attempt to fix travis deploying to production on branches

### LINKED ISSUE #51: Travis always deploys to production, even on pull requests
Probably not the best idea unless they get merged...

### PR DESCRIPTION
**Pull request by Kapin**
Fixes #51

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,11 +20,7 @@ script:
   - ./tools/flow check --lib src/static/js/libs/flow/
   - fab test
 after_success:
-  - openssl aes-256-cbc -K $encrypted_a0fbdc5ae723_key -iv $encrypted_a0fbdc5ae723_iv -in .travis/deploy_key.pem.enc -out .travis/deploy_key.pem -d
-  - chmod 600 .travis/deploy_key.pem
-  - eval "$(ssh-agent)"
-  - ssh-add .travis/deploy_key.pem
-  - fab travis_deploy
+  - bash -x deploy.sh
```
Comment (Trinovantes) [inline .travis.yml]: can this script be put in `/scripts/` folder?

Comment (Kapin) [inline .travis.yml]: yes

**Comment by Trinovantes** [comment]
:+1:

**Comment by ConnorCimowsky** [comment]
LGTM!

**Pull request merged**